client/deis.py

```diff
@@ -800,15 +804,20 @@ def auth_login(self, args):
         password = args.get('--password')
         if not password:
             password = getpass('password: ')
+        disable_ssl_verify = args.get('--disable_ssl_verify')
```

Comment:
for consistency with other projects, `--ssl-verify=false` or `--tlsverify=false` seems to be the common theme.

https://github.com/berkshelf/berkshelf/blob/5d9fae311d4580e3d29027c86ad8e1ba83cb0362/lib/berkshelf/cli.rb#L188-L191
https://github.com/docker/docker/blob/8b95ad230e2ee76450ceb9a80aa8e942a56bb397/docker/docker.go#L93